src/main/java/com/github/nicholasmoser/audio/FFmpeg.java

```diff
@@ -17,11 +17,22 @@ public class FFmpeg {
    * @throws IOException If an I/O error occurs.
    */
   public static String prepareSoundEffect(Path input, Path output) throws IOException {
-    if (!Files.isRegularFile(Paths.get("ffmpeg.exe"))) {
-      throw new IOException("ffmpeg.exe cannot be found.");
+    String ffmpeg;
+    if (System.getProperty("os.name").startsWith("Windows")) {
```

Comment:
Can you replace this with `Platform.isWindows()`? I use that API elsewhere in the codebase and would prefer to keep it consistent.

Reply:
Changed it now

Reply:
Hmm, just realized that the `jna` dependency is test scope currently:

`testImplementation 'net.java.dev.jna:jna:5.9.0'`

so it's only being used in tests. I'll fix this in a follow-up PR

Reply:
https://github.com/NicholasMoser/GNTool/pull/100